docs/node-configuration.md

```diff
@@ -185,27 +185,29 @@ where:
 - `UnlockWallet` contains wallet settings to retrieve account to sign requests to
   NeoFS. Without this setting, the module will use randomly generated private key.
   For configuration details see [Unlock Wallet Configuration](#Unlock-Wallet-Configuration)
-- `Addresses` is a list of NeoFS storage nodes addresses.
-- `Timeout` is a timeout for a single request to NeoFS storage node.
-- `ContainerID` is a container ID to fetch blocks from.
+- `Addresses` is a list of NeoFS storage nodes addresses. This parameter is required.
+  for BlockFetcher functioning.
```

Comment:
`NeoFS BlockFetcher` but at the same time in `network/server` we do have `blockFetcher`.

Reply:
`NeoFS BlockFetcher` is a name of the service. `blockFetcher` is a name of unexported field.

Reply:
Everywhere in the documentation it's `BlockFetcher` If it's not, then documentation must be fixed.